celsos_bookSearch/celsos_bookSearch/View/SearchViewController.swift

```diff
@@ -9,24 +9,64 @@ import UIKit
 
 class SearchViewController: UIViewController {
 
+    // MARK: - Outlets
+    @IBOutlet weak var searchBarOutlet: UISearchBar!
+    @IBOutlet weak var recentSearchesTableViewOutlet: UITableView!
+
```

Comment:
As it was added the Mark for outlets, it is possible to add them for anything you want. 
The attached image is possible alternative to split the code and make it more readable
![Screen Shot 2021-05-21 at 17 16 09](https://user-images.githubusercontent.com/45231134/119193618-41dfb380-ba58-11eb-9e4f-1d8129002859.png)